steward/src/cellars/cellar_v2.rs

```diff
@@ -145,6 +145,28 @@ pub fn get_encoded_call(function: StrategyFunction, cellar_id: String) -> Result
 
             Ok(CellarV2Calls::SetupAdaptor(call).encode())
         }
+        SetPlatformFee(params) => {
+            log_cellar_call(
+                CELLAR_NAME,
+                &SetPlatformFeeCall::function_name(),
+                &cellar_id,
+            );
+
+            if params.platform_fee != 0 {
```

Comment:
Where were we intending to set the platform fee to zero? Additionally, this would need to be tied to specific cellar addresses, we don't want to be able to set the platform fee to zero on any cellar.